nservicebus/handlers/handler-ordering.md

```diff
@@ -21,10 +21,12 @@ The inference here is that the remaining handlers (not specified in the order) a
 
 <!-- import HandlerOrderingWithCode -->
 
-### Specifying First with ISpecifyMessageHandlerOrdering
+If you have been using `ISpecifyMessageHandlerOrdering` in a separate assembly scanned by NServiceBus we suggest to use `INeedInitialization`
```

Comment:
Not sure it is. Imagine people are using `ISpecifyMessageHandlerOrdering` in drop-in assemblies. Now that this interface is gone they are suddenly lost and might not remember to use `INeedInitialization`. The doco on this page hints them to that.